src/modules/CircularGradient/info.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "Circular Gradient",
+    "description": "Gives a circular gradient of the image",
+    "inputs": {
+        "fillColor": {
```

Comment:
And let's remove this parameter here, we'll be able to do all color work in the next module!

Reply:
Just reminding that we should remove this whole parameter, down to line 10!